benchmark/README.md

```diff
@@ -0,0 +1,15 @@
+# Benchmark
```

Comment:
Maybe run it on the M2 Pro `dev-mini` host we have a include the run information at the end of readme? Including host and guest macOS versions (should be 14.4.1).

Reply:
Added in https://github.com/cirruslabs/tart/pull/769/commits/f0adb8b016a09eba67f185ae639888683824e223.